### DISCUSSION TITLE
Lost password to one of the accounts

### DISCUSSION THREAD

**Discussion by Perchant76** [dani-garcia/vaultwarden#3789] [Q&A]
Hello guys!
I host my own instance of vaultwarden and my friend created account on the site, but after some time he forgot the password for the vaulwarden, as I'm am the provider of the instance am I able to somehow retrieve the password or change/reset it for him?

Note: I found that I can use /admin , but I it isn't enabled and I have no idea how to enable it.
Have a great day!

**Comment by kpfleming** [comment]
The Vaultwarden docs have plenty of information about enabling the [admin panel](https://github.com/dani-garcia/vaultwarden/wiki/Enabling-admin-page). Have you checked there?

**Reply by Perchant76** [reply]
Yes I did, I was able to enable admin panel, but after putting in the password as it was: 'password' and in the docker compose file it was ADMIN_TOKEN='password' and it didn't work when put in the Admin panel

**Comment by BlackDex** [comment]
If the user lost his password, and does not have an emergency contact provided, then he is out of luck 

Only way is if still has access to a locked vault, which he can unlock and then export that.

Or of he is part of an organization, which has password reset enabled, and he is enrolled into that, then an org admin can reset his password.

**Comment by stefan0xC** [comment]
> am I able to somehow retrieve the password or change/reset it for him?

No. Unless either of you have taken some precautions there is no way to change a password. You can only delete and recreate his account.

Precautions you could take for the next time, so it's possible then:

a) He can invite you (or someone else) as an [emergency contact](https://bitwarden.com/help/emergency-access/).

b) You could invite him to an organization which has the account [recovery policy](https://bitwarden.com/help/account-recovery/) enabled. Note that "users must be enrolled (either through [self-enrollment](https://bitwarden.com/help/account-recovery/#self-enroll-in-account-recovery) or using the [automatic enrollment policy option](https://bitwarden.com/help/account-recovery/#automatic-enrollment))" to use that feature.

**Comment by Perchant76** [comment]
Well I don't have any of those so I can't do anything about this issue

Closing, can't be solved
